compact/hcal/hcal_barrel_gdml.xml

```diff
@@ -0,0 +1,7682 @@
+      <triangular vertex1="Mesh2Tess_12" vertex2="Mesh2Tess_8" vertex3="Mesh2Tess_13" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_12" vertex2="Mesh2Tess_10" vertex3="Mesh2Tess_8" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_13" vertex2="Mesh2Tess_8" vertex3="Mesh2Tess_5" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_5" vertex2="Mesh2Tess_8" vertex3="Mesh2Tess_6" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_4" vertex2="Mesh2Tess_2" vertex3="Mesh2Tess_1" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_8" vertex2="Mesh2Tess_4" vertex3="Mesh2Tess_15" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_8" vertex2="Mesh2Tess_15" vertex3="Mesh2Tess_6" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_7" vertex2="Mesh2Tess_8" vertex3="Mesh2Tess_10" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_7" vertex2="Mesh2Tess_2" vertex3="Mesh2Tess_4" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_7" vertex2="Mesh2Tess_4" vertex3="Mesh2Tess_8" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_12" vertex2="Mesh2Tess_3" vertex3="Mesh2Tess_7" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_12" vertex2="Mesh2Tess_7" vertex3="Mesh2Tess_10" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_9" vertex2="Mesh2Tess_4" vertex3="Mesh2Tess_1" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_9" vertex2="Mesh2Tess_11" vertex3="Mesh2Tess_4" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_0" vertex2="Mesh2Tess_15" vertex3="Mesh2Tess_4" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_0" vertex2="Mesh2Tess_4" vertex3="Mesh2Tess_11" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_5" vertex2="Mesh2Tess_6" vertex3="Mesh2Tess_15" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_5" vertex2="Mesh2Tess_15" vertex3="Mesh2Tess_0" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_11" vertex2="Mesh2Tess_9" vertex3="Mesh2Tess_14" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_13" vertex2="Mesh2Tess_5" vertex3="Mesh2Tess_0" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_13" vertex2="Mesh2Tess_0" vertex3="Mesh2Tess_11" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_3" vertex2="Mesh2Tess_12" vertex3="Mesh2Tess_13" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_14" vertex2="Mesh2Tess_3" vertex3="Mesh2Tess_13" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_14" vertex2="Mesh2Tess_13" vertex3="Mesh2Tess_11" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_2" vertex2="Mesh2Tess_3" vertex3="Mesh2Tess_14" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_2" vertex2="Mesh2Tess_7" vertex3="Mesh2Tess_3" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_2" vertex2="Mesh2Tess_9" vertex3="Mesh2Tess_1" type="ABSOLUTE"/>
+    <triangular vertex1="Mesh2Tess_2" vertex2="Mesh2Tess_14" vertex3="Mesh2Tess_9" type="ABSOLUTE"/>
+  </tessellated>
+  </solid>
+	
+  <solid name="OuterHCalTile01" material="EJ200" unit="mm" x="0.0" y="0.0" z="0.0">
```

Comment:
They can!  I'll need to get the STEP files from the engineers and then regenerate the gdml, but I have a way to do this in FreeCAD (which is how I did the plates).   I held off on this because I was working on the geometry and overlap issues.  

Will request the STEP files and get started on this, but it may have a timescale that goes into next week.

Reply:
Have the STL source, I can work on this over the weekend.

Reply:
After consulting with Wouter, updated readout to:

`  <readouts>
    <readout name="HcalBarrelHits">
      <segmentation type="NoSegmentation"/>
      <id>system:8,sector:6,tower:6</id>
    </readout>
`